faq/tests/settings/19.py

```diff
@@ -0,0 +1,107 @@
+import os
+from os.path import expanduser
+
+
+USE_TZ = True
+
+TIME_ZONE = "Africa/Johannesburg"
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "NAME": "jmbo",
+        "USER": "postgres",
+        "PASSWORD": "",
+        "HOST": "",
+        "PORT": "",
+    }
+}
+
+xDATABASES = {
```

Comment:
Found out from Hedley, he uses it to comment out settings by renaming the setting since it is faster than commenting every line. Will remove from my settings